packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -219,15 +219,11 @@ export default defineComponent({
         )
       }
 
-      const roles = LinkShareRoles.list(
+      return LinkShareRoles.list(
         this.highlightedFile.isFolder,
-        this.capabilities.files_sharing?.public?.can_edit
+        this.capabilities.files_sharing?.public?.can_edit,
```

Comment:
Let's introduce capability composables for these two capabilities (look for `useCapability...` for examples).